scripts/image-builder/build.sh

```diff
@@ -336,23 +341,41 @@ if [ -z "${IMAGE_VERSION}" ] ; then
     exit 1
 fi
 
-# Download openshift local RPM packages (noarch for python and selinux packages)
-rm -rf openshift-local 2>/dev/null || true
-
-OCP_REPO_NAME="rhocp-4.13-for-rhel-${OSVERSION}-$(uname -m)-rpms"
-reposync -n -a "${BUILD_ARCH}" -a noarch --download-path openshift-local \
-    --repo="${OCP_REPO_NAME}" \
-    --repo="fast-datapath-for-rhel-${OSVERSION}-${BUILD_ARCH}-rpms" >/dev/null
+title "Ensure there is at least one osbuild worker running"
+sudo systemctl start osbuild-composer.socket
+sudo systemctl start "osbuild-worker@1.service"
```

Comment:
I'm not sure this belongs here. If there's some problem with cleanup, maybe we should add code to the `cleanup.sh` script to make sure it starts the service?